src/cf-modals/src/Modal.js

```diff
@@ -0,0 +1,57 @@
+/* ==========================================================================
+   Modal Organism
+   ========================================================================== */
+
+'use strict';
+
+const Organism = require( 'atomic-component/src/components/Organism' );
+
+const ModalOrganism = Organism.extend( {
+
+  insideClicked: false,
+
+  classes: {
+    isVisible: 'o-modal__visible'
+  },
+
+  events: {
+    'click .o-modal_close': 'onModalCloseClick',
```

Comment:
Should values in `key: val` pairs within an object all be left aligned to match cfgov-refresh?